src/services/EdFi.DataManagementService.Api.Tests.E2E/Features/Resources/CreateResourcesValidation.feature.disabled

```diff
@@ -0,0 +1,141 @@
+# This is a rough draft feature for future use.
+@ignore
+Feature: Resources "Create" Operation validations
+
+        Background:
+            Given the Data Management Service must receive a token issued by "http://localhost"
+              And user is already authorized
+
+        @ignore
+        Scenario: Verify new resource can be created successfully
+             When a POST request is made to "/ed-fi/absenceEventCategoryDescriptors" with
+                  """
+                    {
+                        "codeValue": "Sick Leave",
+                        "description": "Sick Leave",
+                        "effectiveBeginDate": "2024-05-14",
+                        "effectiveEndDate": "2024-05-14",
+                        "namespace": "uri://ed-fi.org/AbsenceEventCategoryDescriptor",
+                        "shortDescription": "Sick Leave"
+                    }
+                  """
+             Then it should respond with 201
+              And the record can be retrieved with a GET request
```

Comment:
Please add the location header as in the PUT request. It is actually more important on the POST than on the PUT request.

Reply:
Added